src/main/java/edu/wpi/first/gradlerio/deploy/roborio/FRCJREArtifact.java

```diff
@@ -46,5 +59,21 @@ private boolean jreMissing(DeployContext ctx) {
         return ctx.execute("if [[ -f \"/usr/local/frc/JRE/bin/java\" ]]; then echo OK; else echo MISSING; fi").getResult().contains("MISSING");
     }
 
-
+    private boolean jreOutOfDate(DeployContext ctx) {
+        if (!checkJreVersion) {
+            return false;
+        }
+        String version = getDependency().get().getVersion();
+        CommandDeployResult cmdResult = ctx.execute("opkg list-installed | grep openjdk");
+        if (cmdResult.getExitCode() != 0) {
+            ctx.getLogger().log("JRE not found");
+            return false;
```

Comment:
Should this return true, or is no JRE covered elsewhere?

Reply:
No JRE is checked elsewhere. It runs first, then this runs.